mcstatus/tests/test_async_pinger.py

```diff
@@ -32,22 +33,27 @@ def test_handshake(self):
 
         assert self.pinger.connection.flush() == bytearray.fromhex("0F002C096C6F63616C686F737463DD01")
 
-    def test_read_status(self):
+    def test_read_status(self, monkeypatch):
```

Comment:
In all of the other tests, we're using `unittest.patch` for monkeypatching, we should probably stick to that instead of this pytest specific method of doing it. That said, it would probably make sense to switch to this method, but that's probably not something for this PR to do.

Applies for all uses of pytest's monkeypatch.

Reply:
Changed in d4a1b75c5546df27335ad479df98e38fbc86881d.